### PR TITLE
Support multi runners for services

### LINKED ISSUE #232: Change container name in runner lib
The container name is currently using the service hash as a name. It must be replaced by the instance or runner hash.

### PR DESCRIPTION
**Pull request by antho1404**
fix #232

We can now run multiple instances of the same service without having any conflicts in the container created

### CODE REVIEW

File: packages/runner/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 #### Improvements
 #### Bug fixes
 
+- [#](https://github.com/mesg-foundation/js-sdk/pull/) Fix docker container names to support multi instances fix #232
```
Comment (antho1404) [inline packages/runner/CHANGELOG.md]:
```suggestion
- [#235](https://github.com/mesg-foundation/js-sdk/pull/235) Fix docker container names to support multi instances fix #232
```

**Review by NicolasMahe — CHANGES_REQUESTED** [review]
You need it change the network also:
https://github.com/mesg-foundation/js-sdk/blob/bcd6a6cede969a471fc105a02a29d9baa827b263/packages/runner/src/providers/container/index.ts#L48-L51
and in volume name:
https://github.com/mesg-foundation/js-sdk/blob/bcd6a6cede969a471fc105a02a29d9baa827b263/packages/runner/src/providers/container/index.ts#L186-L195

**Comment by NicolasMahe** [comment]
Not related to this PR but can be fixed here:
The alias of the service on the engine's network seems wrong:
https://github.com/mesg-foundation/js-sdk/blob/bcd6a6cede969a471fc105a02a29d9baa827b263/packages/runner/src/providers/container/index.ts#L102
The alias should simply not be set.

The variable `serviceEndpoint` is really confusing. It should be `engineEndpoint` and directly include the port:
https://github.com/mesg-foundation/js-sdk/blob/bcd6a6cede969a471fc105a02a29d9baa827b263/packages/runner/src/providers/container/index.ts#L81

**Comment by antho1404** [comment]
good catch, fixed

**Review by NicolasMahe — APPROVED** [review]

**Pull request merged**